### DISCUSSION TITLE
rpc username/password now default blank environment variables

### DISCUSSION THREAD

**Discussion by skywayskase** [haugene/docker-transmission-openvpn#2598] [General]
Is seems that in the recent update to the Docker image, `TRANSMISSION_RPC_USERNAME` and `TRANSMISSION_RPC_PASSWORD` are now default environment variables with black values. I didn't have those explicitly called out in my docker config and instead has set my RPC credentials via the settings.json file, but after the recent update, the values in my config file have been overwritten to **blank** values, opening up my transmission WebUI to the world. I had to redo my docker config to explicitly call out those two environment variables to secure my instance. This was pretty upsetting to discover, and I've looked around and have seen any announcemenst/notices regarding this change.

**Comment by skywayskase** [comment]
Well, after some deeper digging, I guess this was a change from several months ago. Now I'm confused about how/why my creds persisted all this time since I do keep my image updated regularly

**Comment by wildoracle** [comment]
How would this open up your transmission WebUI to the world unless you explicitly opened up those ports on your router?
I too am wondering what I'm supposed to enter for these values as this isn't part of the simple docker compose file, and I'm getting this error: 
`/etc/openvpn/start.sh: line 208: TRANSMISSION_RPC_USERNAME: unbound variable`
